docs/src/components/pricing/WhatToExpect.tsx

```diff
@@ -1,114 +1,66 @@
-              perpetuity. There are no further charges unless you choose to renew support and
-              maintenance to cover newer versions. Please note that while the use of the software is
-              perpetual, support and maintenance are not. We roll bug fixes, performance
-              enhancements, and other improvements into new releases; we don&apos;t patch, fix, or
-              in any way alter older versions.
-            </Typography>
-          </Paper>
+    <Container sx={{ py: { xs: 4, md: 8 } }}>
+      <Grid container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, md: 12 }}>
+        <Grid item xs={6}>
+          <Typography variant="h2">What to expect from our available plans</Typography>
         </Grid>
-        <Grid item xs={12} sm={6}>
-          <Paper variant="outlined" sx={{ p: 2, height: '100%' }}>
-            <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
-              <ReplayRoundedIcon fontSize="small" color="primary" />
-              <Typography
-                fontWeight="bold"
-                component="h3"
-                color="text.primary"
-                variant="body2"
-                sx={{ ml: 1 }}
-              >
-                Renewal
+        <Grid item xs={6}>
+          <Stack spacing={2}>
+            <Paper variant="outlined" sx={{ p: 2 }}>
+              <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
+                <AllInclusiveOutlinedIcon fontSize="small" color="primary" />
+                <Typography
+                  fontWeight="bold"
+                  component="h3"
+                  color="text.primary"
+                  variant="body2"
+                  sx={{ ml: 1 }}
+                >
+                  Perpetual license in production
+                </Typography>
+              </Box>
+              <Typography variant="body2" color="text.secondary">
+                With your purchase, you are granted a license to use a version of the product
+                forever in a production environment. There are no further charges unless you choose
+                to renew support and maintenance to cover newer versions. Please note that while the
+                use of the software is perpetual in production, it&apos;s not perpetual for
+                development, so if you&apos;d like to keep making changes to your product,
+                you&apos;ll need to renew your license. We roll bug fixes, performance enhancements,
+                and other improvements into new releases; we don&apos;t patch, fix, or in any way
+                alter older versions.
               </Typography>
-            </Box>
-            <Typography variant="body2" color="text.secondary">
-              If you wish to be able to update to the latest versions and access support after the
-              end of your support period, you have the option to renew support and maintenance.
-              Renewal is priced at 50% off the first-year license cost.
```

Comment:
I guess it could still be interesting to mention the renewals, maybe by renaming it subscription and by how much the price can change one year after the other? Not sure.